fedora_messaging/cli.py

```diff
@@ -155,3 +164,120 @@ def consume(exchange, queue_name, routing_key, callback, app_name):
                 str(e.reason),
             )
             sys.exit(e.exit_code)
+
+
+@cli.command()
+@click.option("--publish-exchange", help=_publish_exchange_help)
+@click.option("--file", help=_publish_file_help, required=True, type=click.File("r"))
+def publish(publish_exchange, file):
+    """
+    Send messages from provided file to an AMQP queue.
+    """
+    msgs_json_str = file.read()
+
+    try:
+        messages = loads(msgs_json_str)
+    except exceptions.ValidationError as e:
+        _log.error("Unable to validate message: %s", str(e))
+        sys.exit(1)
+    except KeyError as e:
+        _log.error("Unable to create message. Missing attribute %s", str(e))
+        sys.exit(1)
+    except ValueError as e:
+        _log.error("Unable to load serialized message: %s", str(e))
+        sys.exit(1)
+
+    for msg in messages:
+        _log.info("Sending message with topic %s", msg.topic)
```

Comment:
Rather than logging I think you want to use https://click.palletsprojects.com/en/7.x/api/#click.echo